### DISCUSSION TITLE
Support for wavpack (via SMB share)

### DISCUSSION THREAD

**Discussion by Soukyuu** [music-assistant/support#2971] [Feature requests and ideas]
Most of my offline music is in wavpack format (.wv + .wvc), sadly it doesn't seem MA detects it as music despite using ffmpeg as a backbone (at least from what I understood). The format uses APEv2 tags. The wvc is the correction file which should be picked up by ffmpeg automatically when opening the .wv file, so only .wv would have to be added to the hardcoded list of extentions I saw something about.

It's the last missing piece to use MA for everything for me, I've been using LMS + squeezelite for offline music.

**Comment by OzGav** [comment]
Try the latest 2.40rc6 or later as the formats accepted have been expanded.
